File: app/src/main/java/com/letianpai/robot/time/parser/custom/CustomClockInfo.java
package com.letianpai.robot.time.parser.custom;

public class CustomClockInfo {

    private int code;
    private CustomWatchConfig data;
    private String msg;

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public CustomWatchConfig getData() {
        return data;
    }

    public void setData(CustomWatchConfig data) {
        this.data = data;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    @Override
    public String toString() {
        return "{" +
                "code:" + code +
                ", data:" + data +
                ", msg:'" + msg + '\'' +
                '}';
    }
}

File: app/src/main/java/com/letianpai/robot/time/parser/custom/CustomClockInfo.kt
package com.letianpai.robot.time.parser.custom

class CustomClockInfo {
    var code: Int = 0
    var data: CustomWatchConfig? = null
    var msg: String? = null

    override fun toString(): String {
        return "{" +
                "code:" + code +
                ", data:" + data +
                ", msg:'" + msg + '\'' +
                '}'
    }
}
